client/common/Button.stories.jsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import { action } from '@storybook/addon-actions';
-import { boolean, text } from '@storybook/addon-knobs';
 
 import Button from './Button';
 import { GithubIcon, DropdownArrowIcon, PlusIcon } from './icons';
 
 export default {
   title: 'Common/Button',
-  component: Button
+  component: Button,
+  args: {
+    children: 'this is the button',
+    label: 'submit',
+    disabled: false
+  }
 };
 
-export const AllFeatures = () => (
-  <Button
-    disabled={boolean('disabled', false)}
-    type="submit"
-    label={text('label', 'submit')}
-  >
-    {text('children', 'this is the button')}
+export const AllFeatures = (args) => (
```

Comment:
The controls on this story don't seem to do anything because we aren't passing down all of the `args`.  Looks like we can [write the story as an object](https://storybook.js.org/docs/react/writing-stories/args) with the new "Component Story Format 3".  Very cool.

```
export default {
  title: 'Common/Button',
  component: Button
}

export const AllFeatures = {
  args: {
    children: 'this is the button'
  }
};
```

Reply:
Thanks for investigating linting... I mistakenly thought these were linting issues on the main branch. I wasn't giving them much thought. I'll try your suggestion later today and see if I can revert them.

Reply:
Yeah with stories you can set some defaults for all stories. Customise default args per story. Or present them with fixed values too if you like. 

Previously this `AllFeatures` story had dynamic args as knobs and all the reset were fixed.

This is a button story from a personal project where args were overridden.
```
const Template: Story<ButtonProps> = (args) => <Button {...args} />;

export const FullWidth = Template.bind({});
FullWidth.args = {
  primary: true,
  children: 'Button',
  fullWidth: true
};
```